From 02b53aad4c970408cab2b76408dbac85e08f8ae9 Mon Sep 17 00:00:00 2001
From: Bernhard Reutner-Fischer <rep.dot.nop@gmail.com>
Date: Sat, 01 Aug 2009 09:21:52 +0000
Subject: ioperm is not used internally, remove hidden_proto

Without a hidden_def we'd end up with __GI_ioperm which is wrong.

Signed-off-by: Bernhard Reutner-Fischer <rep.dot.nop@gmail.com>
---
diff --git a/libc/sysdeps/linux/common/ioperm.c b/libc/sysdeps/linux/common/ioperm.c
index 64fc647..bca2fff 100644
--- a/libc/sysdeps/linux/common/ioperm.c
+++ b/libc/sysdeps/linux/common/ioperm.c
@@ -13,8 +13,10 @@
 
 /* psm: can't #include <sys/io.h>, some archs miss it */
 extern int ioperm(unsigned long __from, unsigned long __num, int __turn_on) __THROW;
+/* Not needed internally;
 libc_hidden_proto(ioperm)
-
+*/
 _syscall3(int, ioperm, unsigned long, from, unsigned long, num, int, turn_on)
+/*libc_hidden_def(ioperm)*/
 
 #endif
--
cgit v0.8.2.1
